setup.py

```diff
@@ -11,6 +11,8 @@
     'Jinja2',
     'docopt',
     'gunicorn',
+    'mechanicalsoup',
```

Comment:
I understood your aim of streamlining your code with ``webob``. This is why I did not recommend using the ``requests.Session()`` object directly in ``net.py``. It would have been much easier but it would not have provided the nice features ``webob`` provides.

So I think that, yes, we should put ``mechanical_soup`` and ``requests` in another requirement list as it is only necessary for authentication.